components/PeerReviews/PeerReviewPerson.tsx

```diff
@@ -0,0 +1,103 @@
+import AuthorAvatar from "~/components/AuthorAvatar";
+import { AuthorProfile } from "~/config/types/root_types";
+import { css, StyleSheet } from "aphrodite";
+import { ReactElement } from "react";
+import icons from "~/config/themes/icons";
+import colors from "~/config/themes/colors";
+import ReactTooltip from "react-tooltip";
+import { ID } from "~/config/types/root_types";
+
+type Props = {
+  id: ID,
+  status?: string,
```

Comment:
We can also create types for these such in such a way that the field is a string-literal field rather than a pure string. This way, it prevents bugs from dumb mistakes like casing or typo
```suggestion
  status?: "INVITED" | "ACCEPTED" | "DECLINED"
```

Reply:
ah yes, that is better. thanks for the tip